### PR TITLE
Added working deployed manifest with few code fixes

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/tab-conversations/csharp/TabConversation/note.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Remove below images for Images folder
```
Comment (ChetanSharma-msft) [inline samples/tab-conversations/csharp/TabConversation/note.txt:1]: Read this text file and act on it.

Don't forget to remove it.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

File: samples/tab-conversations/csharp/TabConversation/note.txt (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/tab-conversations/csharp/TabConversation/note.txt]: Done, removed them

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**